ui/tempo-plugin/src/plugins/tempo-datasource-types.tsx

```diff
@@ -0,0 +1,30 @@
+import { RequestHeaders } from '@perses-dev/core';
+
+export interface HTTPProxy {
```

Comment:
ah probably we should move this to the package `core` so we don't need to duplicate it from the `PrometheusDatasource`

But it can be done in a different PR

Reply:
Updated in the latest commit. I moved HTTPProxy into `http-proxy.ts` in ui/core.